Smartpad-macOS/Smartpad-macOS/Controller/PairViewController.swift

```diff
@@ -13,18 +13,14 @@ class PairViewController: NSViewController{
 
     
     @IBOutlet weak var pairingLabel: NSTextField!
-    var connectionManager: ConnectionManager!
     var deviceList: [String] = []
     var selectedDevice = ""
-    var connData: ConnectionData!
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        connectionManager = ConnectionManager()
-        connData = ConnectionData()
-        connectionManager.pairVC = self
-
+        ConnectionManagerAccess.connectionManager.pairVC = self
```

Comment:
This just makes the ConnectionManager singleton? If so, makes sense

Reply:
That's right